web/packages/teleport/src/services/integrations/types.ts

```diff
@@ -92,6 +93,24 @@ export function getStatusCodeDescription(
   }
 }
 
+export type ExternalCloudAudit = {
```

Comment:
Can we rename as much as possible to ExternalAuditStorage? I will be going back to rename most of the Go code

Reply:
I've renamed all instances in the frontend code and also updated the endpoint paths. Also the context `userAcl` field.